php/getOrder.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * This example script gets information on products and on its related articles from the Tradebyte REST API.
```

Comment:
Description wrong - here we get orders not articles

Reply:
Corrected on 0afb26abf53e3882e3499f97dabdb5dd511c6ed2